### PR TITLE
Add printing of the C4 parameters

### PR DESCRIPTION
**Pull request by pengfeili1**
Add printing of the C4 parameters, even it is verbose but could help users to check whether the results are correct. Jason, you can try to create an additional variable in the command line of add12_6_4 to turn on/off the verbose output. I suggested to turn it on in default due to even it is verbose it is better to be careful here: since messing up the C4 terms could mess up the whole simulations afterwards.

### CODE REVIEW

File: parmed/tools/add1264.py [hunk 1]
```diff
@@ -157,6 +157,17 @@ def params1264(parm, mask, c4file, watermodel, polfile, tunfactor):
                     # There are two C4 terms need to add together between two
                     # different ions
                     result[idx] += c4 / WATER_POL * pol * tunfactor
+
+    # Print the C4 parameters
+    print('Print the C4 parameters (shown with only one decimal place) : ')
+    for i in range(0, ntypes):
```
Comment (swails) [inline parmed/tools/add1264.py]: The 0 in range is redundant, so you can replace this with just range(ntypes). This is what is done everywhere else in ParmEd.

**Comment by swails** [comment]
This printout looks the same as what printLJMatrix provides. This change is not API-friendly (nobody likes library calls to pollute stdout), and seems not to print out *all* atom types that are assigned (since one LJ type usually corresponds to multiple atom types). 

If people are interested in this information, I think they should use printLJMatrix instead. Can you check that command and make sure it satisfies this need?

**Comment by pengfeili1** [comment]
Hi Jason, the change could print out all atom types that are assigned. But I think your printLJMatrix is better, I checked it and found it satisfies this need.

**Comment by swails** [comment]
Excellent.  I'll go ahead and close this PR then.

**Comment by hainm** [comment]
@pengfeili1 
can you replace `for i in range(0, ntypes):` to `for i in range(ntypes):` to make the style consistent in ParmEd?

**Comment by pengfeili1** [comment]
@hainm Thank you Hai. Jason has also suggested me that. Due to printLJMatrix support similar function, this change will not be applied to the code.

**Pull request closed without merging**